extensions/interactions/NumberWithUnits/directives/NumberWithUnits.js

```diff
@@ -135,9 +136,40 @@ oppia.directive('oppiaShortResponseNumberWithUnits', [
   }
 ]);
 
-// Rule evaluation for number with units (will be implemented in M1.3).
+// Rule evaluation for number with units.
 oppia.factory('numberWithUnitsRulesService', [
   'NumberWithUnitsObjectFactory', 'FractionObjectFactory',
   function(NumberWithUnitsObjectFactory, FractionObjectFactory) {
+    var isEqualValues = function(answer, inputs) {
+      if (answer.type === 'fraction') {
+        return answer.fraction.toString() === inputs.fraction.toString();
```

Comment:
This should be done in a similar manner as that of 'isExactlyEqualTo' rule of fractions interaction.

Reply:
Yes but we need to check the type first. If type is fraction, I changed it to string and then compared. In fraction, dict is compared. Both works fine.